include/jemalloc/internal/prof_inlines_b.h

```diff
@@ -5,6 +5,9 @@
 #include "jemalloc/internal/sz.h"
 #include "jemalloc/internal/thread_event.h"
 
+prof_tdata_t *prof_tdata_init(tsd_t *tsd);
+prof_tdata_t *prof_tdata_reinit(tsd_t *tsd, prof_tdata_t *tdata);
```

Comment:
Why not keep them in prof_extern.h? I don’t see why moving them into _inlines.h is better.

Reply:
As I mentioned in the earlier comment, there's no effective difference, in the sense that these functions are still exposed out. The only difference is for better understanding by first time readers: it appears clearer that these functions are only tied to these `prof_inlines_*` files rather than needed externally.

However, if it's not worthwhile to do so, I'm open to putting them back.

Reply:
I worry about the order of the headers (e.g. externs are included first); also the inline functions may change. I don’t think there are other inline headers doing this, so let’s not create an exception here.